capstone/src/test/java/com/google/sps/AndroidManifestParserTest.java

```diff
@@ -66,5 +69,81 @@ public void testIfBytesConvertsToXmlAndReturnsPermissions() {
         permissionsList.add("BIND_DEVICE_ADMIN");
         Assert.assertEquals(permissionsList, ManifestParser.decompressXML(bFile));
     }
+
+    @Test
+    //This test if decompress XML gets attribute tags and can read each position
+    public void testIfByteIsConvertedToXMLTagPosition () {
+        String filePath = Resources.getResource("AndroidManifest.xml").getPath();
+        System.out.print(filePath);
+        AndroidManifestParser ManifestParser = new AndroidManifestParser();
+        File file = new File(filePath);
+        FileInputStream fileInputStream = null;
+        byte[] bFile = new byte[(int) file.length()];
+         try{
+            //convert file into array of bytes
+            fileInputStream = new FileInputStream(file);
+            fileInputStream.read(bFile);
+            fileInputStream.close();
+        }
+        catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
Much of this could be separated into a helper function that returns the populated byte[]. That would help reduce duplicated code in all the tests.